### DISCUSSION TITLE
How to pull GGUF model from Huggingface for Ollama?

### DISCUSSION THREAD

**Discussion by essovius** [open-webui/open-webui#1626] [Q&A]
![image](https://github.com/open-webui/open-webui/assets/7668614/bf7fa6f4-9047-4b9f-b8e1-49ae32600217)

Hi, I'm pretty new with all these, how can I pull this GGUF model, what should I do after this step?

**Comment by rob-x-ai** [comment]
Click on the upload button on the right side of the url you have enter.
